_layouts/default.html

```diff
@@ -88,14 +81,19 @@ <h2>It looks like you have JavaScript disabled!</h2>
       </div>
     </div>
 
+    <div id="consentCookie"></div>
     <script crossorigin="anonymous" integrity="sha512-C5x7QiS49O4ubITbqnAX93oOLk9/cASa+m8GawJMki1i4z8AjFDeklRCqiBBjj7niQULNW6nDhqf7l06zrZ9mA==" src="https://cdnjs.cloudflare.com/ajax/libs/jets/0.14.1/jets.min.js"></script>
     <script crossorigin="anonymous" integrity="sha512-bLT0Qm9VnAYZDflyKcBaQ2gg0hSYNQrJ8RilYldYQ1FxQYoCLtUjuuRuZo+fjqhx/qtq/1itJ0C2ejDxltZVFg==" src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
     <script crossorigin="anonymous" integrity="sha512-nkVN47hpd3LOCD7BXeONKhZVrx2EE69VNvOwH28U6QTfjZ/LBfUN6EkmsinvuGVD5UWhyQeJudkVkkT+B4ffrA==" src="https://cdn.jsdelivr.net/npm/vanilla-lazyload@15.1.1/dist/lazyload.min.js"></script>
+    <script crossorigin="anonymous" integrity="sha512-yXXqOFjdjHNH1GND+1EO0jbvvebABpzGKD66djnUfiKlYME5HGMUJHoCaeE4D5PTG2YsSJf6dwqyUUvQvS0vaA==" src="https://cdn.jsdelivr.net/npm/cookieconsent@3.1.1/build/cookieconsent.min.js"></script>
+    <script async src="https://www.googletagmanager.com/gtag/js?id=G-SMXGN4C672&aip=1"></script>
```

Comment:
Script should be retrived when consent is given.

Reply:
There's no reason to do this other than just a preference. When the script loads, it's started in `consent mode` and the IPs are anonymized, so other than the initial HTTP request to load the script, nothing else happens.

Reply:
Except that we don't get that nice green sheild in Firefox and Safari 🛡️ 😁 
The less is collected by default, the better.